tests/integration/test_error_handling.py

```
@@ -0,0 +1,655 @@
+        binary_content = b"KEY=value\x00\x01\x02\x03\nBINARY=\xff\xfe\xfd"
+        (work_dir / ".env").write_bytes(binary_content)
+
+        config_content = """\
+[encryption]
+backend = "dotenvx"
+"""
+        (work_dir / "envdrift.toml").write_text(config_content)
+
+        env = {"PYTHONPATH": integration_pythonpath}
+
+        result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)